components/feature/app-links/README.md

```diff
@@ -0,0 +1,20 @@
+# [Android Components](../../../README.md) > Feature > App-Links
+
+A session components that intercepts URLs and checks if they are openable via another app.
+
+### Setting up the dependency
+
+Use Gradle to download the library from [maven.mozilla.org](https://maven.mozilla.org/) ([Setup repository](../../../README.md#maven-repository)):
+
+```Groovy
+implementation "org.mozilla.components:feature-app-links:{latest-version}"
+```
+
+## Usage
+
```

Comment:
nit: Looks like "Setting up the dependency" moved above "Usage" here :)

Reply:
👍